packages/cmake/Modules-CMakeDetermineSystem.cmake.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Modules/CMakeDetermineSystem.cmake b/Modules/CMakeDetermineSystem.cmake
+index dc208c600..f3ec4da2d 100644
+--- a/Modules/CMakeDetermineSystem.cmake
++++ b/Modules/CMakeDetermineSystem.cmake
+@@ -43,7 +43,7 @@ if(CMAKE_HOST_UNIX)
+     else()
+       exec_program(${CMAKE_UNAME} ARGS -r OUTPUT_VARIABLE CMAKE_HOST_SYSTEM_VERSION)
+     endif()
+-    if(CMAKE_HOST_SYSTEM_NAME MATCHES "Linux|CYGWIN.*|Darwin|^GNU$")
++    if(CMAKE_HOST_SYSTEM_NAME MATCHES "Linux|CYGWIN.*|Darwin|^GNU$|Android")
```

Comment:
The branch below calls uname -p, which sets CMAKE_HOST_SYSTEM_PROCESSOR to unknown just like on linux, so use this one instead.